code/modules/mob/living/carbon/human/human.dm

```diff
@@ -159,7 +159,7 @@
 		if(!prob(martial_art.deflection_chance))
 			return ..()
 		if(!src.lying && dna && !dna.check_mutation(HULK)) //But only if they're not lying down, and hulks can't do it
-			src.visible_message("<span class='warning'>[src] deflects the projectile!</span>", "<span class='userdanger'>You deflect the projectile!</span>")
+			src.visible_message("<span class='danger'>[src] deflects the projectile, there's no way you'll be able to hit them with ranged weapons!</span>", "<span class='userdanger'>You deflect the projectile!</span>")

```

Comment:
I considered that too, but the message makes sense. You see some dude deflect automatic weapon fire with his bare hands and it makes sense that your internal observation would be "Oh shit there's no way I'm going to be able to shoot that guy" even if you're not the one doing the shooting. I wanted to strike a balance between "The game is explicitly telling you what their status is" and a more roleplay vibe of "this is what I'm observing with my eyes".